Automattic/vip-go-mu-plugins
# Parsely 2.6: prevent warnings

## Description

Change default from null to empty array to prevent warnings thrown by array_diff on line 1743 and any subsequent functions with array as a typehint


## Changelog Description

### Parsely 2.6 fix warnings from failed array_diff typehint

Defaulting to null starts a cascade of errors from functions with array typehints

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Visit any site with Parsely enabled
2. Use Query Monitor or similar to view PHP Errors
3. Verify no `array_diff(): Argument #2 must be of type array, null` or `array_intersect_key(): Argument #2 must be of type array, null` errors occur
